mkosi/manifest.py

```diff
@@ -214,6 +216,17 @@ def record_pkg_packages(self) -> None:
                 self.source_packages[source] = source_package
             source_package.add(package)
 
+    def record_extension_release(self) -> None:
+        if self.context.config.output_format not in (OutputFormat.sysext, OutputFormat.confext):
+            return
+
+        with complete_step("Recording sysext / confext information in manifest…"):
+            type_ = "sysext" if self.context.config.output_format == OutputFormat.sysext else "confext"
+            d = "usr/lib" if self.context.config.output_format == OutputFormat.sysext else "etc"
+            p = self.context.root / d / f"extension-release.d/extension-release.{self.context.config.output}"
+
+            self.extensions.append((type_, read_env_file(p)))
```

Comment:
Please use `+= [...]` instead of `append()` like we do everywhere else.

Reply:
Yes. Done.